conf/sim.py

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env python
+# SPDX-License-Identifier: Apache-2.0
+# Copyright(c) 2019 Intel Corporation
+
+from scapy.contrib.gtp import *
```

Comment:
Dockerfile show do pip install `scapy` instead of the current commit based zip file